### ISSUE TITLE
Accidental sticker press wipes draft

### ISSUE THREAD

**Issue by ghubkiller** [signalapp/Signal-Android#9868]
- [x ] I have searched open and closed issues for duplicates
- [x ] I am submitting a bug report for existing functionality that does not work as intended
- [x ] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When typing a message sometimes I accidently hit a sticker this clears my whole typed out message (draft).
This problem has also occured to me when recipient has messaged me during typing also.

### Steps to reproduce
-type message
-select a sticker
-draft is wiped
**Actual result:** draft wipes
**Expected result:** draft doesn't wipe

### Link to debug log
https://debuglogs.org/27d6bf8b18816637b79a1aca4c6ea7f86b94d00f7e88feb5ff752f40709b54bb

**Comment by 0xGREG** [comment]
Can confirm this is an annoying bug and should get fixed

**Comment by alan-signal** [comment]
I could not reproduce clearing of a draft with Signal Stickers.

I also cannot see how selecting a Signal Sticker would be a common accident as would require two taps.

So maybe you are talking about a Sticker from your keyboard? Which keyboard are you using?

**Comment by ghubkiller** [comment]
Hi,
I'm using gboard. I actually downloaded the beta yesterday and touch wood I
haven't had an issue with it since. I removed all the stickers so I didn't
have a problem. I have also tried reproducing this fault with the beta and
it works seamlessly.

It happened numerous times in the past, maybe on and off for a few months.
Whether it either send a sticker or a large emoji.

Kind Regards,
Ben Aldridge
